contrib/verify_ffi.py

```
@@ -68,48 +46,26 @@ def main():
 
-    headers_changed = headers_result.returncode != 0
-    docs_changed = docs_result.returncode != 0
-
-    if headers_changed or docs_changed:
+    if docs_result.returncode != 0:
+        print()
+        print("FFI documentation is out of date!\n")
+        print("Documentation changes detected:")
+        subprocess.run(
+            ["git", "--no-pager", "diff", "--",
+             "key-wallet-ffi/FFI_API.md", "dash-spv-ffi/FFI_API.md"],
```

Ruff (0.15.6)

[error] Starting a process with a partial executable path

(S607)